rse-py-scripting.Rmd

```diff
@@ -103,42 +133,461 @@ optional arguments:
   -h, --help  show this help message and exit
 ```
 
-and issues errors when users give the program invalid arguments:
+As the `--help` hint shows,
+`argparse` can handle [long options][long-option-shell] as well as traditional single-letter options;
+the online help has examples of this and much more.
+
+## How can I configure common options for a program? {#rse-py-scripting-configure}
+
```

Comment:
In #156 (and #223) there is some discussion about whether we want to include any content on configuration (i.e. beyond command line options). We never really settled on an answer to that question - this PR basically says that we do want to include it.

Reply:
I re-read Taschuk's paper, and I think we do need some discussion of persistent configuration for reproducibility - she has a good rant about shell scripts that rely on defaults, but then defaults change.